pkgs/development/tools/analysis/smatch/default.nix

```diff
@@ -8,19 +8,36 @@
 , buildc2xml ? false
 , libllvm
 , libxml2
+, substituteAll
+, llvmPackages
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "smatch";
   version = "1.73";
 
   src = fetchFromGitHub {
     owner = "error27";
     repo = "smatch";
-    rev = version;
-    sha256 = "sha256-Pv3bd2cjnQKnhH7TrkYWfDEeaq6u/q/iK1ZErzn6bME=";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Pv3bd2cjnQKnhH7TrkYWfDEeaq6u/q/iK1ZErzn6bME=";
   };
 
+  patches = [
+    (let
+      clang-major = lib.versions.major (lib.getVersion llvmPackages.clang-unwrappedclang);
+      clang-lib = lib.getLib llvmPackages.clang-unwrappedclang;
```

Comment:
```suggestion
      clang-major = lib.versions.major (lib.getVersion llvmPackages.clang);
      clang-lib = lib.getLib llvmPackages.libclang;
```

Reply:
Bad copy-paste in last minute refactoring, my bad.

Works with `llvmPackages.clang-unwrapped` but not `llvmPackages.clang`